pandas/io/sql.py

```diff
@@ -77,6 +78,16 @@ def _is_sqlalchemy_connectable(con):
         return False
 
 
+def _gt14() -> bool:
```

Comment:
i think we have a test decorator for this type of thing right?

Reply:
we have a decorator to skip a test depending on the version, not to re-write the test